app/configurations/config.default.js

```diff
@@ -865,4 +865,7 @@ export default {
   },
 
   showAlternativeLegs: true,
+
+  // Notice! Turning on this setting forces the search for car routes.
```

Comment:
When I first read this comment, I thought this magically turned on the car itinerary suggestions, which this doesn't do. Make it more explicit that this causes a car search used only for the co2 comparison purposes.

Reply:
Yes, comment was meant to indicate the query is run, not that the car option is shown on ui. Added a clarification.